tests/dash/dash_acl.py

```diff
@@ -0,0 +1,384 @@
+        self.dash_config_info = dash_config_info
+        self.inner_extra_conf = inner_extra_conf
+        self.expected_receiving = expected_receiving
+        self.description = description + "_" + str(self.inner_extra_conf)
+
+    def get_description(self):
+        return self.description
+
+
+class AclTestCase(object):
+    def __init__(self, duthost, dash_config_info):
+        __metaclass__ = abc.ABCMeta  # noqa: F841
+        self.duthost = duthost
+        self.dash_config_info = dash_config_info
+        self.test_pkts = []
+
+    @abc.abstractmethod
+    def config(self):
+        pass
+
+    @abc.abstractmethod
+    def teardown(self):
+        pass
+
+
+class DefaultAclGroupTest(AclTestCase):
+    def __init__(self, duthost, dash_config_info):
+        super(DefaultAclGroupTest, self).__init__(duthost, dash_config_info)
+        self.acl_group = DEFAULT_ACL_GROUP
+        self.group = AclGroup(self.duthost, self.acl_group, self.dash_config_info[ENI])
+
+    def config(self):
+        self.group.bind(DEFAULT_ACL_STAGE)
+
+    def teardown(self):
+        self.group.unbind()
+        del self.group
+
+
+class AclRuleTest(AclTestCase):
+    def __init__(self, duthost, dash_config_info):
+        super(AclRuleTest, self).__init__(duthost, dash_config_info)
+        self.rule_confs = []
+
+    def add_rule(self, rule_conf):
+        rule_conf[ACL_RULE] = self.__class__.__name__ + "_" + rule_conf[ACL_RULE]
+        apply_acl_config(self.duthost, ACL_RULE_TEMPLATE, rule_conf, op="SET")
+        self.rule_confs.append(rule_conf)
+
+    def add_test_pkt(self, test_pkt):
+        test_pkt.description = self.__class__.__name__ + "_" + str(len(self.test_pkts) + 1) + "_" + test_pkt.description
+        self.test_pkts.append(test_pkt)
+
+    def teardown(self):
+        for rule_conf in self.rule_confs:
+            apply_acl_config(self.duthost, ACL_RULE_TEMPLATE, rule_conf, op="DEL")
+        self.rule_confs = []
+
+
+class DefaultAclRule(AclRuleTest):
+    def __init__(self, duthost, dash_config_info):
+        super(DefaultAclRule, self).__init__(duthost, dash_config_info)
+        self.acl_group = DEFAULT_ACL_GROUP
+
+    def config(self):
+        self.add_rule({
+            ACL_GROUP: self.acl_group,
+            ACL_RULE: "allow",
+            ACL_PRIORITY: 1,
+            ACL_ACTION: "allow",
+            ACL_TERMINATING: "true",
+            ACL_PROTOCOL: "17",
+            ACL_SRC_ADDR: "0.0.0.0/0",
+            ACL_SRC_PORT: "1234"
+        })
+
+
+class AclPriorityTest(AclRuleTest):
+    def __init__(self, duthost, dash_config_info):
+        super(AclPriorityTest, self).__init__(duthost, dash_config_info)
+        self.acl_group = DEFAULT_ACL_GROUP
+        self.src_ip = "10.0.0.2"
+        self.src_ip_prefix = self.src_ip + "/32"
+
+    def config(self):
+        self.add_rule({
+            ACL_GROUP: self.acl_group,
+            ACL_RULE: "deny_10",
+            # TODO. This priority should be lower than rule2's (2)
```

Comment:
@Pterosaur can you please update the priorities in this file? Now everything should work as expected.